plugin/src/main/resources/io/jenkins/plugins/analysis/warnings/groovy/ParserConfiguration/help-consoleLogScanningPermitted.html

```diff
@@ -0,0 +1,14 @@
+<div>
+  <p>
+    Console Log parsing has to be processed directly on the Jenkins controller
+    which means running the parser groovy code (configured here) directly on the controller which <em>CAN BE UNSAFE</em> from a security point of view.
+  </p>
+    e.g. if your groovy code is poorly written.
```

Comment:
Is this sentence intended to be in the middle of 2 `<p>` tags?

![Bildschirmfoto 2021-06-29 um 11 11 28](https://user-images.githubusercontent.com/503338/123770801-dc8aa800-d8ca-11eb-85ef-b72fa27ed7ca.png)

Reply:
🤷 It was deliberate, yes ... and I know that the resulting message renders ok, but I don't know whether or not it would be better to have that middle sentence within its own `<p>` ... `</p>` section too.
i.e. deliberate ... but potentially misguided 😉